Dockerfile

```diff
@@ -40,4 +40,9 @@ ENV ANSIBLE_SSH_PIPELINING=true
 ENV ANSIBLE_LOCAL_TMP=/tmp
 ENV ANSIBLE_REMOTE_TMP=/tmp
 
+RUN git clone https://github.com/reismarcelo/sastre-ansible.git /tmp/sastre-ansible && \
+    ansible-galaxy collection build /tmp/sastre-ansible/cisco/sastre --output-path /tmp/sastre-ansible && \
+    ansible-galaxy collection install -f /tmp/sastre-ansible/cisco-sastre-1.0.16.tar.gz && \
+    rm -fr /tmp/sastre-ansible
+
```

Comment:
Will this `cisco-sastre` module be eventually publicly available so we can remove this part?